addons/web/static/src/views/relational_model.js

```diff
@@ -2640,10 +2640,12 @@ export class Group extends DataPoint {
     }
 
     exportState() {
-        return {
+        const state = {
             isFolded: this.isFolded,
             listState: this.list.exportState(),
         };
+        delete state.listState.countLimit;
+        return state;
```

Comment:
It does not seem like the proper fix to do here, so please do not change the relational model, the countLimit was added to the list state in a recent fix: https://github.com/odoo/odoo/commit/07aa7b0af09a8405836e30ffa2dc449fca665676#diff-e22194e5de98a436b4aa85bff663d42a7b94fcf294f0259649dcac4198d22753

instead do:
```patch
diff --git a/addons/web/static/src/views/list/list_renderer.js b/addons/web/static/src/views/list/list_renderer.js
index 57a5b82e3728..1070114dcc01 100644
--- a/addons/web/static/src/views/list/list_renderer.js
+++ b/addons/web/static/src/views/list/list_renderer.js
@@ -934,7 +934,7 @@ export class ListRenderer extends Component {
         return {
             offset: list.offset,
             limit: list.limit,
-            total: list.count,
+            total: group.count,
             onUpdate: async ({ offset, limit }) => {
                 await list.load({ limit, offset });
                 this.render(true);
@@ -1654,7 +1654,7 @@ export class ListRenderer extends Component {
     }
 
     showGroupPager(group) {
-        return !group.isFolded && group.list.limit < group.list.count;
+        return !group.isFolded && group.list.limit < group.count;
     }
 
     toggleGroup(group) {
```